pom.xml

```diff
@@ -46,7 +47,7 @@
         <zookeeper.version>3.4.9</zookeeper.version>
         <hadoop.version>2.5.2</hadoop.version>
         <hbase.hadoop.version>2.5.2</hbase.hadoop.version>
-        <kafka.version>0.10.2.1</kafka.version>
+        <kafka.version>1.0.0</kafka.version>
```

Comment:
Why `1.0.0` and not a geomesa version?

Reply:
most people should be using 1.0.0 at this point - we've kept geomesa back because 1) we'd have to file cqs, and 2) we still use 0.9 in some other projects. i tested the quickstart with 0.9, 0.10 and 1.0.0 so should be ok